backend/routes/forms/index.py

```diff
@@ -31,3 +32,19 @@ async def get(self, request: Request) -> JSONResponse:
         return JSONResponse(
             forms
         )
+
+    @requires(["authenticated", "admin"])
+    async def post(self, request: Request) -> JSONResponse:
+        form_data = await request.json()
+        try:
+            form = Form(**form_data)
+        except ValidationError as e:
+            return JSONResponse(e.errors())
+
+        if await request.state.db.forms.find_one({"_id": form.id}):
+            return JSONResponse({
+                "error": "Form with same ID already exists."
```

Comment:
Let's use this diff as an opportunity to kaizen this fix.
```suggestion
                "error": "id_taken"
```